package.json

```diff
@@ -23,7 +23,7 @@
   },
   "dependencies": {
     "clipboard": "^2.0.1",
-    "leaflet": "^1.3.1",
+    "leaflet": "^1.3.4",
```

Comment:
Turns out this doesn't solve the problem, but updating to the latest version doesn't hurt anything.